src/shadowsocks_config.spec.ts

```diff
@@ -60,6 +59,25 @@ describe('shadowsocks_config', () => {
       }
     });
 
+    it('normalizes IPv6 address hosts', () => {
```

Comment:
is it worth adding serialization/parsing tests with unexpanded IPv6 hosts?

Reply:
Done (parsing only, it doesn't seem to make sense for serialization).